clouditor-engine-core/src/main/java/io/clouditor/discovery/Scan.java

```diff
@@ -59,12 +58,7 @@
    * The asset type, this scan is targeting. This is automatically parsed from the {@link
    * ScannerInfo}.
    */
-  @JsonProperty
-  @ManyToOne
-  @JoinColumn(name = "type_value", nullable = false)
-  @MapKey
-  @Id
-  private AssetType assetType = new AssetType();
+  @Id @JsonProperty private String assetType;
```

Comment:
Don't you lose the relation to the AssetType?

Reply:
Yes but this is intended, since the asset types are just used as ids here. Also the asset type objects do not exist yet at the time the scans are loaded. If we change that, we would need to change too much in the UI so we should keep it as a string for now.